assets/js/Containers/FrevaGPT/utils.js

```diff
@@ -27,4 +27,9 @@ export const objectToQueryString = (obj) => {
     return Object.entries(obj)
       .map(([key, value]) => `${encodeURIComponent(key)}=${encodeURIComponent(value)}`)
       .join('&');
-  };
\ No newline at end of file
+  };
+
+export const truncate = (value) => {
```

Comment:
* `const` instead of `var`
* `.substring` instead of `substr` (deprecated)

Instead of this function you could also use the bootstrap css-class `text-truncate` which also truncates the text but only visually in CSS and leaves the text as-is in HTML. It looks a bit different, though. You don't need to do this, just a remark.

Reply:
Adapted see: https://github.com/FREVA-CLINT/freva-web/pull/60/commits/7c1b41db7a6a093a7a2e785a9be8bff07a81a83b

I skipped the usage of the CSS class for now because it removes to much of the string. But that is personal preference. I will keep it in mind